Godot_test/ClientScripts/GameLevel.cs

```diff
@@ -22,6 +22,7 @@ public partial class GameLevel : Node
 	string pathToEnemies = "res://Sprites/LivingEntities/Enemies/";
 	public static List<GodotGame.Entities.Enemy> enemyList = new();
```

Comment:
We shall check again why we can not have also Dictionary here as we have for other types